docs/stable_boxes.md

```diff
@@ -75,16 +75,16 @@ Packer will create an image by bootstrapping an operating system from a kickstar
 
 #### Example publishing workflow
 
-For example, here is the workflow for `centos7-katello-devel-stable`:
+For example, here is the workflow for `centos8-katello-devel-stable`:
 
-1. A cron job or scheduled automation builds the box image with Packer using the `packer/centos7-katello-devel-stable.json` template.
+1. A cron job or scheduled automation builds the box image with Packer using the `packer/centos8-katello-devel-stable.json` template.
```

Comment:
Found that we still have the file named `packer/centos7-katello-devel-stable.json` there 🤔

Reply:
@chris1984 said he was working on that. Not sure how far he got.

Reply:
I will open a PR with the new json files, I got them working now making devel boxes